internals/daemon/response.go

```diff
@@ -169,22 +150,26 @@ func (f fileResponse) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	http.ServeFile(w, r, string(f))
 }
 
+func Reponsef(status int, format string, v ...interface{}) Response {
```

Comment:
As per the proposal, we can now use this from a derivative project using ```http.<error>``` values.